routes/patientsRoutes.js

```diff
@@ -0,0 +1,17 @@
+/* 
+Name: Dylan Bellinger
+Date: 11/11/2024 
+Description: Patient routes for endpoints.
+*/
+var express = require('express');
+var router = express.Router();
+const patientController = require('../controller/patient.controller');
```

Comment:
Shouldn't this be `const {getAllPatients,
     getPatientBannerInfo,
     getPatientByID,
     createPatient,
     updatePatient,
     deletePatient} =  require('../controller/patient.controller');` ?